suprkewl-bot/ext/util.py

```diff
@@ -265,7 +267,9 @@ def save():
                 return [fname, fp]
 
             fname, fp = await save()
-
+        channel = ctx.author.voice.channel
+        context = await channel.connect()
+        context.play(discord.FFmpegPCMAudio(fname))
```

Comment:
This has a few problems:
The author is not always in a VC.
The bot may not have permission to speak in or connect to the VC.
The bot may already be playing music there or somewhere else - then what?

I think that the user can also just set up a soundboard...